tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(SOF_FUZZER C)
+
+include(ExternalProject)
+
+set(source_directory "${PROJECT_SOURCE_DIR}/../testbench")
+
+ExternalProject_Add(sof_tb
+	SOURCE_DIR "${source_directory}"
+	PREFIX "${PROJECT_BINARY_DIR}/sof_tb"
+	BINARY_DIR "${PROJECT_BINARY_DIR}/sof_tb/build"
+	CMAKE_ARGS -DBUILD_LIBRARY=ON
+		-DCMAKE_INSTALL_PREFIX=${PROJECT_BINARY_DIR}/sof_tb/build
+		-DCMAKE_VERBOSE_MAKEFILE=${CMAKE_VERBOSE_MAKEFILE}
+	BUILD_COMMAND make
+	BUILD_ALWAYS 1
+	BUILD_BYPRODUCTS "${PROJECT_BINARY_DIR}/sof_tb/build/lib/libsof_tplg_parser.so"
+)
+
+add_library(sof_testbench SHARED IMPORTED)
```

Comment:
It's not full testbench, but just tplg parser build so, name like `tplg_parser` may be a better fit.